packages/cli/src/commands/init.ts

```diff
@@ -147,17 +147,24 @@ export default class Init extends Command {
     project: ProjectSpecBase,
     projectPath: string,
     flags: {npm: boolean; 'install-dependencies': boolean}
-  ): Promise<void> {
-    const [defaultEndpoint, defaultAuthor, defaultDescription] = await readDefaults(projectPath);
-
-    project.endpoint = !Array.isArray(defaultEndpoint) ? [defaultEndpoint] : defaultEndpoint;
-    const userInput = await input({
-      message: 'RPC endpoint:',
-      default: defaultEndpoint[0] ?? 'wss://polkadot.api.onfinality.io/public-ws',
-      required: false,
-    });
-    if (!project.endpoint.includes(userInput)) {
-      (project.endpoint as string[]).push(userInput);
+  ): Promise<{isMultiChainProject: boolean}> {
+    const {
+      author: defaultAuthor,
+      description: defaultDescription,
+      endpoint: defaultEndpoint,
+      isMultiChainProject,
+    } = await readDefaults(projectPath);
+
+    if (!isMultiChainProject) {
+      project.endpoint = !Array.isArray(defaultEndpoint) ? [defaultEndpoint] : defaultEndpoint;
+      const userInput = await input({
+        message: 'RPC endpoint:',
+        default: defaultEndpoint[0] ?? 'wss://polkadot.api.onfinality.io/public-ws',
```

Comment:
I think we can remove the fallback endpoint, the chances of it being for polkadot network are slim
```suggestion
        default: defaultEndpoint[0],
```